vhdbuilder/packer/install-dependencies.sh

```diff
@@ -353,41 +353,41 @@ done
 
 # calico images used by AKS
 CALICO_CNI_IMAGES="
-v3.8.9
-v3.8.9.1
+3.8.9
```

Comment:
why the change

Reply:
this is just for consistencies sake, all the versions don't start with v for the other dependencies, so wanted to be consistent here, nothing serious ;)